File: backend/src/main/java/com/bremen/backend/domain/user/dto/UserResponse.java
package com.bremen.backend.domain.user.dto;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class UserResponse {
	private String username;
	private String nickname;
	private String introduce;
	private String profileImage;
	private int followerCnt;
	private int followCnt;
}

File: bremen/src/types/UserResponse.ts
interface LoginResponse {
  status: number;
  message: string;
  item: {
    profileImage: string;
    nickname: string;
    refreshToken: string;
    accessToken: string;
    id: number;
    name: string;
    username: string;
  };
}

interface ICheck {
  status: number;
  message: string;
  data: string;
}

interface ISignUpResponse {
  status: number;
  message: string;
  item: {
    username: string;
    nickname: string;
    introduce: null | string;
    profileImage: null | string;
    followerCnt: number;
    followCnt: number;
    follow: boolean;
  };
}
export type {LoginResponse, ICheck, ISignUpResponse};
